docs/layouts/types/grid.md

```diff
@@ -1,3 +1,7 @@
+---
+description: Maximize dashboard space with the grid layout in Node-RED Dashboard 2.0, offering flexible organization.
```

Comment:
Not sure this description is a good fit.

The benefits of grid layout are more about flexibility and responsive layouts for adapting to different screen sizes.

Reply:
```suggestion
description: Create flexible and responsive layouts using a grid system for Node-RED Dashboard 2.0.
```